MatrixSDKTests/MatrixSDKTestsData.h

```diff
@@ -91,7 +91,7 @@ FOUNDATION_EXPORT NSString * const kMXTestsAliceAvatarURL;
 
 
 #pragma mark - mxAlice
-@property (nonatomic, readonly) MXCredentials *aliceCredentials;
+@property (nonatomic, strong, readonly) MXCredentials *aliceCredentials;
```

Comment:
This is more a question for me to learn from: I thought `strong` was the default - is this for consistency, or are there conditions where the defaults can be changed?

Reply:
Yes, this is just for consistency. strong is indeed the default for reference types and assign for value types.